pkg/ddevapp/drupal.go

```diff
@@ -566,15 +566,20 @@ func drupalEnsureWritePerms(app *DdevApp) error {
 	// So we chmod +w the two files that a Drupal install may set read-only
 	// *inside* the container, allowing mutagen access to it again
 	if app.IsMutagenEnabled() {
```

Comment:
IMO instead of returning nil in the not-running condition, it would be better to change the condition to siterunning && IsMutagenEnabled, that way if something gets added in the future after the if stanza it won't cause unexpected behavior.